src/gatsby/config.ts

```diff
@@ -31,7 +31,7 @@ const getPlugins = () => {
           packages,
           apps,
         },
-        excludeExpr: ["default"],
+        excludeExpr: ["default", "secrets.SENTRY_AUTH_TOKEN"],
```

Comment:
This is needed to resolve a warning shown by the templating engine due to the `{{ }}` syntax shown in the example.   Used here:

https://github.com/getsentry/sentry-docs/blob/952411d1f0ec42d30b0b2248cce77815537a3947/src/gatsby/plugins/gatsby-remark-variables/index.js#L42-L45